cnab/app/run

```
@@ -0,0 +1,40 @@
+#!/bin/sh
```

actions should report as failed when they fail, and fail fast

```suggestion
#!/bin/sh

set -o errexit
set -o nounset
set -o pipefail
```